src/th/reapertrans/src/eu/kanade/tachiyomi/extension/th/reapertrans/ReaperTrans.kt

```diff
@@ -0,0 +1,12 @@
+package eu.kanade.tachiyomi.extension.th.reapertrans
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class ReaperTrans : MangaThemesia(
+    "ReaperTrans",
+    "https://reapertrans.com",
+    "th",
+    dateFormat = SimpleDateFormat("MMMM d, yyyy", Locale("en")),
```

Comment:
This line is functionally identical to the default and can be removed.
Don't forget to remove imports as well, otherwise it will complain during lint.

For reference, the default:
```kt
val dateFormat: SimpleDateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale.US),
```